dist/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>My-Todo-list</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+    integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+  <meta property="og:image" content="../screen.png">
+  <meta property="og:description"
+    content="This is a Todo-list project using Webpack">
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+
+<body>
+  <div class="container">
+    <h4 class="title">Today's Todo</h4>
+    <form>
+      <input class="input-todo" type="text" placeholder="Add to your List..." autofocus />
+      <button class="add-btn">
+        <i class="fas fa-location-arrow"></i>
+      </button>
+    </form>
+    <!-- Todos to be added Automatically below -->
+    <div class="all-todos">
+       
+    </div>
+    <div class="clear">
+      <a class="clear-btn">Clear All</a>
```

Comment:
- [ optional] kindly ensure the whole clear all button works, not just when you click on the link(or on the word clear all)